mscore/prefsdialog.cpp

```diff
@@ -250,7 +247,248 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
 
 void PreferenceDialog::start()
       {
+      normalWidgets = vector<PreferenceItem*>{
```

Comment:
OH YES!!! this is soooo much clearer! thanks!

Reply:
:^) I am glad you like it.